src/lsp-error-reporter.cpp

```diff
@@ -86,10 +86,18 @@ lsp_error_formatter::lsp_error_formatter(byte_buffer &output,
 
 void lsp_error_formatter::write_before_message(severity sev,
                                                const source_code_span &origin) {
-  if (sev == severity::note) {
+  int severity_value{};
```

Comment:
Nit: Don't bother initializing severity_value. As written, if we forget to assign to this variable, we can't get a compiler warning.
```suggestion
  int severity_value;
```